graph.py

```diff
@@ -0,0 +1,55 @@
+from collections import defaultdict
+
+class Graph:
+    def __init__(self) -> None:
+        self.adjacency_list = defaultdict(list)
```

Comment:
Benefit of 'defaultdict' ???

Reply:
I need to add extra logic to add the element if I declare with normal dict() logic

Reply:
What's the extra logic?

Reply:
need to check if the key is present already before appending, otherwise I need to add the key with with empty list and then add. It requires few lines of code